plugins/by-name/blink-cmp/default.nix

```diff
@@ -42,4 +42,18 @@ lib.nixvim.plugins.mkNeovimPlugin {
     };
     signature.enabled = true;
   };
+
+  extraOptions = {
+    setupLspCapabilities = lib.nixvim.options.mkEnabledOption "setup LSP capabilities for blink-cmp";
+  };
+
+  extraConfig = cfg: {
+    plugins.lsp.capabilities =
+      lib.mkIf cfg.setupLspCapabilities # lua
+        ''
+          -- Capabilities configuration for blink-cmp
+          capabilities.textDocument.completion.completionItem.snippetSupport = true
```

Comment:
```suggestion
```

Already included in the get_lsp_capabilities
https://github.com/Saghen/blink.cmp/blob/9f32ef5c3bb44f943238bbcde0c467936475f177/lua/blink/cmp/sources/lib/init.lua#L266